### DISCUSSION TITLE
Repositories metrics has changed

### DISCUSSION THREAD

**Discussion by andry81** [lowlighter/metrics#1146] [Q&A] [answered]
https://github.com/andry81-cache/andry81--gh-content-cache/commit/4eef63ed69ca4299358812e7ffe650383562e1fc#diff-566213d7bf64064d44a7e8b161368c146b7d1d07e403356652f97f7aa40abbd3

Changed from 2.21 GB to 75.9 MB.

As I understand, is the new size without forks?

**Comment by lowlighter** [comment, marked as answer]
Hi 👋 !

https://metrics.lecoq.io/andry81?repositories.affiliations= will yield the previous result you had before 👍 

[`repositories_affiliations`](https://github.com/lowlighter/metrics/blob/master/source/plugins/base/README.md#repositories_affiliations) default value was changed to `owner` a long time ago (don't remember when) but maybe you had it generated from a older version

Hope it helps

**Reply by andry81** [reply]
From the docs is not clear what data is included into calculation: `collaborator`, `organization_member` is related to forks or organizations?

If set only the owner `repositories_affiliations=owner`, does all organization repositories where the owner is added would be added into calculation? Does the forks can be added or removed? How the organizations can be added or removed?

**Reply by lowlighter** [reply]
I'm not too sure either, these are direct translations from GitHub API.

From my understanding, OWNER is for repositories hosted on your account, COLLABORATOR is any repository where you have push access and ORGANIZATION_MEMBER is any repository from an organization where you're member (whether or not you have push access or contributed)

Empty removes any constraint and include any repository loosely related to you

Forks are handled by  [`repositories_forks`](https://github.com/lowlighter/metrics/blob/master/source/plugins/base/README.md#repositories_forks) option
